erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py

```diff
@@ -1984,6 +1984,18 @@ def test_item_tax_net_range(self):
 		sales_invoice.save()
 		self.assertEqual(sales_invoice.items[0].item_tax_template, "_Test Account Excise Duty @ 10 - _TC")
 
+	def test_sales_invoice_with_discount_accounting_enabled(self):
```

Comment:
Instead of checking just discount amount, check all GL Entries similar to this test case

https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py#L1730

Reply:
Done.